services/src/northbound/src/main/java/org/openkilda/northbound/utils/RequestCorrelation.java

```diff
@@ -0,0 +1,16 @@
+package org.openkilda.northbound.utils;
+
+/**
+ * Correlation id for every single request request.
+ */
+public final class RequestCorrelation {
+
+    public static final String CORRELATION_ID = "correlation_id";
+
+    private static final ThreadLocal<String> ID = new ThreadLocal<>();
```

Comment:
This should be InheritableThreadLocal to cover cases when we perform async tasks.